assessments/0010-etcd/etcd-implementation.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Implementing etcd Doc Improvements
+tags: etcd
+---
+
+# Introduction
+
+This document provides actionable suggestions for improving the etcd technical documentaiton.  
+
+For an analysis and general discussion of recommendations on etcd technical documentation, see [etcd-analysis.md][etcd-analysis].
+
+## Recommendations, requirements, and best practices
+
+This analysis measures documentation against CNCF project maturity standards, and suggests possible improvements. In most cases there is more than one way to do things. Few recommendations here are meant to be prescriptive. Rather, recommendations are based on documentation best practices as understood by the reviewers. The recommended implementations represent the reviewers' experience with how to apply those best practices. In other words, borrowing terminology from the lexicon of [RFCs][rfc-keywords], the changes described here should be understood as "recommended" or "should" at the strongest, and "optional" or "may" in many cases. Any "must" or "required" actions are clearly denoted as such, and pertain to legal requirements such as copyright and licensing issues. 
+
+The top-level documentation recommendations for this project are:
+
+- Complete and update instructional documentation:
+    - Describe etcd's user roles (personas).
+    - Document Kubernetes and Linux installation procedures.
+    - Update and clarify quick start and new user workflows.
+    - Document key tasks as procedures rather than tutorials.
+    - Clarify which user roles (personas) each task is for.
+    - Going forward, write release notes for each major and minor release.
```

Comment:
Etcd does have release notes in the repository.  Here, are you saying also copy those to the user documentation?